src/main/java/duke/utils/Storage.java

```diff
@@ -0,0 +1,68 @@
+package duke.utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.DukeException;
+import duke.task.Task;
+
+/**
+ * Represents a storage that stores the tasks in a file.
+ */
+public class Storage {
+    private String path;
+    private File file;
+
+    /**
+     * Constructs a Storage object.
+     *
+     * @param path The path of the file to store the tasks.
+     */
+    public Storage(String path) {
+        this.path = path;
+        this.file = new File(this.path);
+    }
+
+    /**
+     * Creates a file to store the tasks if the file does not exist.
+     *
+     * @throws DukeException If the file cannot be created.
+     */
+    public void saveTasks(ArrayList<Task> tasks) throws DukeException {
+        try {
+            FileWriter filewriter = new FileWriter(this.path);
+            for (Task task : tasks) {
+                filewriter.write(task.toSaveString() + "\n");
```

Comment:
I like that you still wrapped the loop body in curly brackets despite there only being one line.